python/xorbits/deploy/docker/entrypoint.sh

```diff
@@ -4,5 +4,5 @@ set -e
 if [[ "$1" == *"/"* ]]; then
   $@
 else
-  /opt/conda/bin/python -m "$1" ${@:2} --log-conf /srv/logging.conf
+  /opt/conda/bin/python -m "$1" ${@:2}
```

Comment:
Maybe we can just delete the "cp /opt/xorbits/python/xorbits/deploy/oscar/file-logging.conf /srv/logging.conf" command in the xorbits/deploy/docker/Dockerfile since we do not use it anymore.